interview-questions.md

```diff
@@ -7,65 +7,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: Rails follows the pattern of MVC (Model, View, Controller); which means it comes default with a structure for a database (model), a web service (Controller), and for web pages(View). You can make a Full-Stack application solely through Rails because of this!
 
-  Researched answer:
+  Researched answer: Model, View, Controller is the pattern Ruby follows, which means it can be it's own full stack application that supplies the model (like a database), the controller (modify and update the databse), and the view (where users can interact with you application).
 
 
 
 2. What does it mean if an application has CRUD? What does CRUD stand for?
 
-  Your answer:
+  Your answer: If an application has CRUD, I can assume it automatically has a persistent storage system, which means it can Create data, Read data, Update data, and Delete data (C.R.U.D.). CRUD is the bare minimum required to create a persistent storage system.  
 
-  Researched answer:
+  Researched answer: If an application has CRUD, it essentially means that it has the capabilities to have a persistent storage system. So you can Create, Read, Update, and Delete items inside a persistent storage system which is the minimum capabilities required to have a persistent storage system.
 
 
 
 3. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: A relational database is a form of database that holds information in tables consisting of rows and columns. The information is stored to show relationships between sets of data inside the rows and columns. There are many different types of databases, some do not follow how the data is related to each other and they do not have to consist of rows and columns.
 
-  Researched answer:
+  Researched answer: Relational databases are a type of database that throw pieces of data that are related to each other onto a table designed with columns and rows. There are many kinds of databases, and you can build them to suit your needs.
```

Comment:
yup. Main thing in web development is that there are relational and non-relation databases. It'd be good to know the difference between these two.